mosip-identity-plugin/pom.xml

```diff
@@ -3,11 +3,17 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
+	<parent>
+		<groupId>io.mosip.esignet.plugin</groupId>
+		<artifactId>esignet-plugin-parent</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+	</parent>
```

Comment:
Should we maintain a common parent, the plugins are more individual right ?

Reply:
Removed parent pom